applications/pelion_client/README.rst

```diff
@@ -289,6 +303,10 @@ Each button press is counted and the count value is reflected onto the respectiv
 
 You can reset Digital Input objects' counters by putting zero into their respective values in the Pelion Device Management Portal.
 
+.. warning::
```

Comment:
Do not use warnings unless there is something that puts the user's health or life at risk: https://projecttools.nordicsemi.no/confluence/display/TECHDOC/Style+and+usage+guide

Instead, use a note.